src/kendo.maskedtextbox.js

```diff
@@ -532,6 +540,26 @@ var __meta__ = { // jshint ignore:line
             return result;
         },
 
+        _blinkInvalidState: function () {
+            var that = this;
+
+            that.element.addClass(STATEINVALID);
+            if(that._validationIcon) {
+                that._validationIcon.show();
+            }
+            clearTimeout(that.invalidStateTimeout);
+            delete that.invalidStateTimeout;

```

Comment:
Is this really necessary? Can setting it to `null` work better? You can also think about prefixing it with an underscore `_invalidStateTimeout`, so it doesn't look like some kind of public thing.